.github/workflows/publish.yaml

```
@@ -0,0 +1,12 @@
+name: Publish CLI Tools
+
+on:
+    push:
+        tags:
+            - 'v[0-9]+\.[0-9]+\.[0-9]+.*' # Matches tags like v1.2.3 and v1.2.3-pre.1
```

actionlint (1.7.4)

character '\' is invalid for branch and tag names. only special characters [, ?, +, *, \, ! can be escaped with \. see `man git-check-ref-format` for more details. note that regular expression is unavailable. note: filter pattern syntax is explained at https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet

(glob)